### PR TITLE
Add Cincinnati and GDS?

### PR DESCRIPTION
**Pull request by plaindocs**
Unsure about where we want to put GDS

### CODE REVIEW

**Comment by ericholscher** [comment]
Looks good. We should link up the AU site now too, since we have a landing page for it I think.

File: docs/include/conf/current.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 - :doc:`Write the Docs Portland </conf/portland/2018/index>`, May 6-8, **Portland, Oregon**
 - :doc:`Write the Docs + Open Help Cincinnati </conf/cincinnati/2018/index>`, August 18-22, **Cincinnati, Ohio**
 - :doc:`Write the Docs Prague </conf/prague/2018/index>`, September 9-11, **Prague, Czech Republic**
-- :doc:`Write the Docs Australia </conf/australia/2018/index>`, November 15-16, **Melbourne, Australia**
+- :doc:`Write the Docs Australia </conf/au/2018/index>`, November 15-16, **Melbourne, Australia**
```
Comment (ericholscher) [inline docs/include/conf/current.rst]: Ha yea, I was just wondering what to do about that.

Comment (plaindocs) [reply docs/include/conf/current.rst]: I kinda think we should use Australia.

Comment (ericholscher) [reply docs/include/conf/current.rst]: Yea, if we wanted to be really consistent, we'd use "Melbourne" :/

**Comment by plaindocs** [comment]
Closed in favor of #507

**Pull request closed without merging**